client/app/components/onboardingGuide/GuideFilterPage.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import scss from './styles.scss';
+
+const GuideFilterPage = (props) => {
+
+  const handleClick = function(e, path) {
+    e.preventDefault();
+    props.changePage(path);
+  }
+
+  return (
```

Comment:
This might be a subjective/stylistic question, but do we really need a className for every single element?